CIP-0027.md

```diff
@@ -0,0 +1,64 @@
+
```

Comment:
Thanks for adding a tentative header with author/title/etc..
(also the tentative file should be added in the proper directory subfolder (/CIP-0027/)